actionview/lib/action_view/template/resolver.rb

```diff
@@ -60,6 +56,7 @@ def build_path_regex
       end
 
       def parse(path)
+        @regex ||= build_path_regex
```

Comment:
I worry this might be a thread safety issue on ex. jruby?

Reply:
Hmm...  More than one thread may call `build_path_regex` before `@regex` is assigned, but it shouldn't matter which one wins since they should all derive the same result, no?